pandemic/record_test.go

```diff
@@ -4,6 +4,41 @@ import (
 	"testing"
 )
 
+func TestProbabilityOfDrawingAlreadyDrawnCard(t *testing.T) {
+
+	gs := GameState{
+		InfectionDeck: InfectionDeck{
+			Drawn: []InfectionCard{
+				InfectionCard{
+					Name: "SanFrancisco",
+				},
+			},
+		},
+		InfectionRate: 3,
+	}
+
+	if prob := gs.ProbabilityOfCity("SanFrancisco"); prob != 0 {
+		t.Fatalf("Should have had a 0%% chance of SanFranciso, got %v", prob)
+	}
+
+}
+
+func TestProbabilityOfDrawingCardAtStart(t *testing.T) {
+
+	gs := GameState{
+		InfectionDeck: InfectionDeck{
+			Drawn: []InfectionCard{},
+		},
+		InfectionRate: 2,
+	}
+
+	// TODO(wjs) how 2 floating point
+	if prob := gs.ProbabilityOfCity("SomeCity"); prob != 0.04166666666666674 {
```

Comment:
We could test against a rounded value if that makes this more aesthetically pleasing